__tests__/pet.test.js

```diff
@@ -7,6 +7,7 @@ describe('constructor', () => {
 });
 
 describe('constructor', () => {
+    // remove any comments prior to committing
```

Comment:
You shouldn't commit comments. The reasoning behind this is that a) if your comment has no purpose then it doesn't serve a purpose being there, and b) good, tested code should describe itself.